sdk/maps/Azure.Maps.Weather/src/Models/StormWindRadiiSummary.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+using Azure.Core.GeoJson;
+using Azure.Maps.Common;
+
+namespace Azure.Maps.Weather.Models
+{
+    /// <summary> Displayed when details=true or radiiGeometry=true in the request. </summary>
+    [CodeGenModel("StormWindRadiiSummary")]
```

Comment:
We don't need `CodeGenModel` if we just want to change its members? `CodeGenModel` is for renaming the class.